.github/workflows/spiced_docker.yml

```diff
@@ -49,11 +49,30 @@ jobs:
           push: ${{ startswith(github.ref, 'refs/tags/v') }}
           build-args: |
             REL_VERSION=${{ env.REL_VERSION }}
+          tags: |
+            ghcr.io/spiceai/spiceai:${{ env.REL_VERSION }}-slim
+            spiceai/spiceai:${{ env.REL_VERSION }}-slim
+            ${{ env.LATEST_GHCR_TAG }}-slim
+            ${{ env.LATEST_DOCKERHUB_TAG }}-slim
+
+      - name: Build and push Docker images (full)
+        id: docker_build_full
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ startswith(github.ref, 'refs/tags/v') }}
+          build-args: |
+            REL_VERSION=${{ env.REL_VERSION }}
+            CARGO_FEATURES=odbc
```

Comment:
Please note, I've decided to keep/re-use CARGO_FEATURES parameter to specify what features we would like to include to full build (more control and flexibility). Alternatively we can have dedicated flag to control this

Reply:
I think eventually we will have a `full` flag to control all of the features that go into the full image - but not needed for now.